src/ASM/ASMs3D.C

```diff
@@ -1535,53 +1535,45 @@ bool ASMs3D::updateCoords (const Vector& displ)
 }
 
 
-void ASMs3D::getBoundaryNodes (int lIndex, IntVec& nodes) const
+void ASMs3D::getBoundaryNodes (int lIndex, IntVec& nodes, int basis) const
 {
+  if (basis == 0)
+    basis = 1;
+
   if (!svol) return; // silently ignore empty patches
 
```

Comment:
Sånn ca en forglemmelse.

-------- Opprinnelig melding --------
Fra: kmokstad <notifications@github.com>
Dato: 03.05.2016 22:29 (GMT+01:00)
Til: OPM/IFEM <IFEM@noreply.github.com>
Ko: Arne Morten Kvarving <Arne.Morten.Kvarving@sintef.no>, Author <author@noreply.github.com>
Emne: Re: [OPM/IFEM] PETSc reboot - general domain decomposition (#7)


In src/ASM/ASMs3D.C<https://github.com/OPM/IFEM/pull/7#discussion_r61950229>:

>  {
> +  if (basis == 0)
> +    basis = 1;
> +
>    if (!svol) return; // silently ignore empty patches
>


for tilsvarende 2D-metode brukes getBasis(basis), mens her går du på svol direkte. Ulogisk, eller forglemmelse? Ellers gjelder de andre kommentarene for 2D-metoden her også.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/OPM/IFEM/pull/7/files/c7217e143f115795decd5a0825082a79366a80f1..42876e3e4e0331f01b4f774f65f6bce431496644#r61950229>